Description: don't try missing NewMockThriftTest test
 Upstream doesn't ship NewMockThriftTest hence don't try to use it.
Author: Laszlo Boszormenyi (GCS) <gcs@debian.org>
Last-Update: 2019-11-30

---

--- thrift-0.13.0.orig/test/go/src/common/clientserver_test.go
+++ thrift-0.13.0/test/go/src/common/clientserver_test.go
@@ -21,14 +21,10 @@ package common
 
 import (
 	"context"
-	"errors"
 	"gen/thrifttest"
-	"reflect"
 	"sync"
 	"testing"
-	"thrift"
 
-	"github.com/golang/mock/gomock"
 )
 
 type test_unit struct {
@@ -61,6 +57,7 @@ func TestAllConnection(t *testing.T) {
 }
 
 func doUnit(t *testing.T, unit *test_unit) {
+/*
 	ctrl := gomock.NewController(t)
 	defer ctrl.Finish()
 	handler := NewMockThriftTest(ctrl)
@@ -81,6 +78,7 @@ func doUnit(t *testing.T, unit *test_uni
 	}
 	defer trans.Close()
 	callEverythingWithMock(t, client, handler)
+*/
 }
 
 var rmapmap = map[int32]map[int32]int32{
@@ -98,6 +96,7 @@ var xxs = &thrifttest.Xtruct{
 var xcept = &thrifttest.Xception{ErrorCode: 1001, Message: "some"}
 var defaultCtx = context.Background()
 
+/*
 func callEverythingWithMock(t *testing.T, client *thrifttest.ThriftTestClient, handler *MockThriftTest) {
 	gomock.InOrder(
 		handler.EXPECT().TestVoid(gomock.Any()),
@@ -333,3 +332,4 @@ func callEverythingWithMock(t *testing.T
 		t.Errorf("Unexpected error in TestVoid() call: %s", err)
 	}
 }
+*/
